tests/bench_sizeof_coretypes/main.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2014 René Kijewski <rene.kijewski@fu-berlin.de>
+ *               2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Print size of core types
+ *
+ * @author      René Kijewski <rene.kijewski@fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "cib.h"
+#include "clist.h"
+#include "panic.h"
+#include "kernel_types.h"
+#include "list.h"
+#include "mbox.h"
+#include "msg.h"
+#include "mutex.h"
+#include "priority_queue.h"
+#include "ringbuffer.h"
+#include "rmutex.h"
+#ifdef MODULE_CORE_THREAD_FLAGS
+#include "thread_flags.h"
+#endif
+#include "thread.h"
+
+
+#define P(NAME) printf("    tcb->%-11s            %3u     %3u\n", #NAME, \
+                       (unsigned)sizeof(((thread_t *) 0)->NAME), \
+                       (unsigned)offsetof(thread_t, NAME));
```

Comment:
Macro shouldn't end in a semicolon

Reply:
agreed, i copy pasted this from the old `tests/sizeof_tcb` app. Will fix.